internal/config/config.go

```
@@ -161,3 +161,8 @@ func LogConfigState() {
 	s, _ := json.MarshalIndent(Vars, "", "  ")
 	log.Printf("[NOTICE] Config State: %s", s)
 }
+
+func AuditDir() string {
```

Create this directory on read if it doesn't exist, ignoring the error if it does.